doc/release/upcoming_changes/17731.deprecation.rst

```diff
@@ -0,0 +1,5 @@
+``compat.npy_load_module`` deprecated
+-------------------------------------
+
+The undocumented ``numpy.compat.npy_load_module`` is deprecated. Use
+``SourcFileLoader`` instead.
```

Comment:
Spelling it out as ``importlib.machinery.SourceFileLoader`` would be good (and fixes a typo).

Reply:
Perhaps with single backticks, so that it links:
```suggestion
`importlib.machinery.SourceFileLoader` instead.
```